en/install-maas.md

```diff
@@ -0,0 +1,308 @@
+OpenStack cloud. 
+
+The following is what you'll find in a typical MAAS environment and  we'll use
+this as the framework for our own deployment:
+
+- A **Region controller** interacts with and controls the wider environment for
+  a region 
+- One or more **Rack controllers** manage locally grouped hardware, usually
+  within a data centre rack
+- Multiple **Nodes** are individual machines managed by the Rack controller,
+  and ultimately, the Region controller
+- Complex **Networking** topologies can me modelled and implemented by MAAS,
+  from a single fabric to multiple zones and many overlapping spaces
+
+## What you'll need
+
+MAAS can work at any scale, from a test deployment using nothing but [LXD
+on a single machine][conjureup] to thousands of machines deployed across
+multiple regions. 
+
+It's our intention to build a useful minimal deployment of OpenStack, capable
+of both performing some real work and scaling to fit more ambitious projects.
+
+To make this minimal configuration as accessible as possible, we'll be
+using single nodes for multiple services, reducing the total number of machines
+required. The four Compute nodes, for instance, will co-host Ceph, Glance and
+Swift, as well as the other services required by OpenStack. 
+
+The hardware we'll be using is based on the following specifications:
+
+- 1 x MAAS Rack with Region controller: 8GB RAM, 2 CPUs, 1 NIC, 40GB storage
+- 1 x Juju node: 4GB RAM, 2 CPUs, 1 NIC, 40GB storage
+- 4 x OpenStack Compute nodes: 8GB RAM, 2 CPUs, 2 NICs, 80GB storage
+
+To get a better idea of the minimum requirements for the machines that run
+MAAS, take a look at the [MAAS documentation][minreq].
+
+As with the hardware, our network topology is also going to be as simple as
+possible whilst remaining both scaleable and functional. It contains a single
+zone for the four Compute nodes, with the machine hosting the MAAS region and
+rack controllers connected to both the external network and the single zone.
+It's recommended that MAAS is the sole provider of DHCP and DNS for the network
+hosting the nodes MAAS is going to manage, but we'll cover this in an imminent
+step. 
+
+Your hardware could differ considerably from the above and both MAAS and Juju
+will easily adapt. The Juju node could operate perfectly adequately with half
+the RAM (this would need to be defined as a bootstrap constraint) and adding
+more nodes will obviously improve performance. 
+
+!!! Note:
+    We'll be using the web UI whenever possible, but it's worth noting that
+    everything (and more) we do with MAAS can also be done from the [CLI][maascli]
+    and the [API][maasapi]. 
+
+## Package installation
+
+The first step is to install [Ubuntu Server 16.04 LTS][ubuntuserver] on the
+machine that's going to host both the MAAS Rack and Region controllers. The
+Ubuntu Server install menu includes the option to [nstall and configure both
+controllers][installcontrollers], but it's easier to install Ubuntu Server
```

Comment:
I'm not sure we want to say its easier. We can refer to the step-by step (is there still one) in the MAAS docs for this, but say that to cover more use cases, we will assume that you have a fresh install of Ubuntu Server and show how to install from packages